Templates/SendGrid-Timer-CSharp/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "defaultFunctionName": "SendGridTimerCSharp",
+    "description": "$SendGridTimerCSharp_description",
```

Comment:
We need a string for $SendGridTimerCSharp_description in https://github.com/Azure/azure-webjobs-sdk-templates/blob/dev/Resources/Resources.resx file.